### PR TITLE
Fix error with menus.

### PR DESCRIPTION
**Pull request by demoneaux**

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ function setWindowFocus (flag) {
...
 function setAllowNav (flag) {
   getMenuItem('Preferences').enabled = flag
-  getMenuItem('Create New Torrent...').enabled = flag
+  getMenuItem(
+    process.platform === 'darwin'
+      ? 'Create New Torrent...'
+      : 'Create New Torrent from Folder...'
+  ).enabled = flag
   var item = getMenuItem('Create New Torrent from File...')
   if (item) item.enabled = flag
```
Comment (mathiasvr) [inline src/main/menu.js]: Nice catch!
I have a suggestion for making it a little more readable, but I don't mind keeping it like this.
```javascript
if (process.platform === 'darwin') {
    getMenuItem('Create New Torrent...').enabled = flag
} else {
    getMenuItem('Create New Torrent from Folder...').enabled = flag
    getMenuItem('Create New Torrent from File...').enabled = flag
}
```

Comment (feross) [inline src/main/menu.js]: Yeah, good catch @demoneaux. Thanks 👍 

I also prefer @mathiasvr's version. Will fix up after merge.

Comment (feross) [inline src/main/menu.js]: https://github.com/feross/webtorrent-desktop/commit/b08d2739968f09685af9fe61bb862acd72762c82

Comment (demoneaux) [inline src/main/menu.js]: :+1:

**Pull request merged**